dependencies.tsv

```diff
@@ -5,9 +5,9 @@ github.com/juju/gojsonschema	git	e1ad140384f254c82f89450d9a7c8dd38a632838	2015-0
 github.com/juju/loggo	git	8477fc936adf0e382d680310047ca27e128a309a	2015-05-27T03:58:39Z
 github.com/juju/names	git	a6a253b0a94cc79e99a68d284b970ffce2a11ecd	2015-07-09T13:59:32Z
 github.com/juju/schema	git	afe1151cb49d1d7ed3c75592dfc6f38703f2e988	2015-08-07T07:58:08Z
-github.com/juju/testing	git	31ee64312c3c64cc94a5b41ea7a200b42e0f9767	2015-09-02T15:44:51Z
-github.com/juju/utils	git	03357ca1fc29e4a2b9189c82906475b6bc34f84a	2015-09-10T07:18:02Z
+github.com/juju/testing	git	5955efc6d5f61e41a13e4409ec6bae7a6975546f	2015-10-10T23:46:57Z
+github.com/juju/utils	git	1fc85d7f4bc5e885cc86f68bb6ac610efee5be38	2015-10-14T03:39:15Z
 golang.org/x/crypto	git	aedad9a179ec1ea11b7064c57cbc6dc30d7724ec	2015-08-30T18:06:42Z
 gopkg.in/check.v1	git	b3d3430320d4260e5fea99841af984b3badcea63	2015-06-26T10:50:28Z
 gopkg.in/mgo.v2	git	f4923a569136442e900b8cf5c1a706c0a8b0883c	2015-08-21T15:31:23Z
-gopkg.in/yaml.v1	git	9f9df34309c04878acc86042b16630b0f696e1de	2014-09-24T16:16:07Z
+gopkg.in/yaml.v2	git	53feefa2559fb8dfa8d81baad31be332c97d6c77	2015-09-24T14:23:14Z
```

Comment:
I wonder if we should standardise on the version of yaml in core? Maybe not important thought? handwave handwave bikeshed etc